File: nodejs/helloworld.js
var http = require('http');

var server = http.createServer(function (request, response) {
  console.log("Handle " + request.method + " " + request.url);
  response.writeHead(200, {"Content-Type": "text/plain"});
  response.end("Hello world from my Node program!\n");
});

server.listen(8000);

console.log("Server running at http://127.0.0.1:8000/");

File: go/helloworld.go
package main

import (
	"fmt"
	"net/http"
)

func handler(w http.ResponseWriter, r *http.Request) {
	fmt.Println("Handle", r.Method, r.URL)
	fmt.Fprintf(w, "Hello world from my Go program! TEST!!!\n")
}

func main() {
	http.HandleFunc("/", handler)
	http.ListenAndServe(":8000", nil)
}
